react-components/src/desktop/messages.js

```diff
@@ -30,6 +30,11 @@ var intlData = {
         emailSupportTooltip: "support@cyverse.org",
         supportChatLink: "Chat With Support",
         supportChatTooltip: "Chat may be disabled by your browser's settings",
+        collaboratorsTooltip: "Manage your private set of collaborators",
```

Comment:
May as well get the `List` terminology in here too:
```suggestion
        collaboratorsTooltip: "Manage your private collaborator lists",
```